Dockerfile

```diff
@@ -51,9 +76,9 @@ RUN pip install --no-cache-dir --upgrade pshtt==0.6.6
 # Install domain-scan
 ###
 RUN git clone https://github.com/18F/domain-scan \
-    ${HOME}/domain-scan/
+    ${CISA_HOME}/domain-scan/
```

Comment:
I could not combine this with my above suggestion because of the GitHub UI and I would rather have the suggestion be completely in one of these comments.

```suggestion
```